### PR TITLE
Update quadratic_equations_complex_numbers.py with One line Easy Equation

### PR DESCRIPTION
**Pull request by SparkScratch-P**
### Describe your change:

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: maths/quadratic_equations_complex_numbers.py [hunk 1]
```diff
@@ -18,10 +18,9 @@ def quadratic_roots(a: int, b: int, c: int) -> tuple[complex, complex]:
 
     if a == 0:
         raise ValueError("Coefficient 'a' must not be zero.")
-    delta = b * b - 4 * a * c
 
-    root_1 = (-b + sqrt(delta)) / (2 * a)
-    root_2 = (-b - sqrt(delta)) / (2 * a)
+    root_1 = (((-b) + ((b**2)-(4*a*c))**5))/(2*a))
```
Comment (poyea) [inline maths/quadratic_equations_complex_numbers.py:22]: Is this correct?

File: maths/quadratic_equations_complex_numbers.py [hunk 2]
```diff
@@ -18,10 +18,9 @@ def quadratic_roots(a: int, b: int, c: int) -> tuple[complex, complex]:
...
+    root_2 = (((-b) - ((b**2)-(4*a*c))**5))/(2*a))
```
Comment (poyea) [inline maths/quadratic_equations_complex_numbers.py:23]: I doubt if this is correct.

Comment (SparkScratch-P) [reply maths/quadratic_equations_complex_numbers.py:23]: It works, tested.

**Comment by SparkScratch-P** [comment]
Yes, there was an error. I have Corrected it. Please Merge.

File: maths/quadratic_equations_complex_numbers.py [hunk 3]
```diff
@@ -18,10 +18,9 @@ def quadratic_roots(a: int, b: int, c: int) -> tuple[complex, complex]:
...
```
[the remark this answers is not in the document]
Comment (cclauss) [reply maths/quadratic_equations_complex_numbers.py:24]: If the proposed solution is faster than the original then we need a [timeit](https://docs.python.org/3/library/timeit.html) (or similar) benchmark to prove it.
* https://github.com/TheAlgorithms/Python/search?q=timeit

**Pull request closed without merging**